src/Restaurant.js

```diff
@@ -1,28 +1,26 @@
 import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
 
 const Restaurant = () => {
   // array destructuring
   // setOrders is a function that allows me to modify the value of the orders variable
   // 0 is the initial value
   const [orders, setOrders] = useState(0);
```

Comment:
Given this is a count, it might be better naming to this to `orderCount` and `setOrderCount` - that would make it more obvious what this is. Developers will be mostly likely to assume a variable called `orders` contains an array of order objects.

Reply:
Hey Douglas, I'm not sure I'm understanding this, correct me if I'm wrong - I could rename the components Restaurant to orderCount and RestaurantButton to setOrderCount? THank you :)

Reply:
No sorry I am referring to `orders` and `setOrders` respectively :)

Reply:
THat makes sense, thanks for clarifying!